force-app/main/default/flows/Create_Event_Registration_Id_Fill_Registration_Fee.flow-meta.xml

```diff
@@ -1,12 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Flow xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>54.0</apiVersion>
+    <assignments>
```

Comment:
@t-gul On the flow, I see the first decision element named "Id_Check". However, the first outcome checks only "canceled" field.

Reply:
@sgoEma Yes, the flow work like that logic.

Reply:
@sgoEma Flow is updated.